src/locale/es/LC_MESSAGES/django.po

```diff
@@ -4106,3 +4106,24 @@ msgstr "Actualizar consejo de prudencia"
 
 msgid "Update the Danger indication"
 msgstr "Actualizar indicacion de peligro"
+
+msgid "Laboratory Room not belongs to this laboratory"
+msgstr "La sala de laboratorio no pertenece al laboratorio"
+
+msgid "Laboratory Room does not belongs to this organization"
+msgstr "La sala de laboratorio no pertenece a la organización"
+
+msgid "Shelfobject does not belongs to this laboratory."
+msgstr "El objecto de estante no pertenece al laboratorio."
+
+msgid "Shelfobject does not belongs to this organization."
+msgstr "El objecto de estante no pertenece a la organización."
+
+msgid "Shelfobject does not belongs to this shelf."
+msgstr "El objecto de estante no pertenece al estante."
+
+msgid "Shelf does not belongs to this laboratory."
+msgstr "El estante no pertenece al laboratorio."
+
+msgid "Shelf does not belongs to this organization."
+msgstr "El estante no pertenece a la organización."
```

Comment:
Since in English we are writing things like "does not belong to this organization.", etc, then in Spanish it should say "no pertenece a **esta** organización", in Spanish we also need to reference which object we are making reference to.